rules/phy-credo.mk

```diff
@@ -1,3 +1,3 @@
 PHY_CREDO = phy-credo_1.0_amd64.deb
-$(PHY_CREDO)_URL = "https://github.com/aristanetworks/sonic-firmware/raw/446f30ccd8626f904d89d5798da7294948e090a6/phy/phy-credo_1.0_amd64.deb"
+$(PHY_CREDO)_URL = "https://github.com/aristanetworks/sonic-firmware/raw/4c6fa09910656b78103416652651b486deaba4aa/phy/phy-credo_1.0_amd64.deb"
```

Comment:
@abohanyang which platform is this fix tested? Is AOC managed by Xcvrd?

Reply:
Hi Prince,

This change addresses two issues:
Issue 1 "Fix an error on platforms with external phy but not using phy-credo service. Some sonic-mgmt tests fail due to this error"
 - We tested this fix against 7800R3-48CQ2-LC and 7800R3-48CQM2-LC and confirmed that phy-credo service exits gracefully on 7800R3-48CQM2-LC.

Issue 2 "Add AOC support for platforms that use the phy-credo service."
- The phy-credo service updates the firmware settings of Credo Serdes based on the transceiver type. The AOC module is still managed by the Xcvrd.
- 7800R3-48CQ2-LC uses phy-credo service. 7800R3-48CQM2-LC does not use phy-credo service. So we tested AOC modules on 7800R3-48CQ2-LC.